### ISSUE TITLE
Markers Duplicates and not stable 

### ISSUE THREAD

**Issue by pretivj** [appoly/ARCore-Location#89]
Markers are duplicate and they are not stable in camera position.
Markers are flickering and rendering so many times.

**Comment by todoranalex** [comment]
If you're creating a ViewRenderable for each marker, then there won't be any duplicates. If you use the same ViewRendarable for all markers in your list, you'll get duplicates.

**Comment by pretivj** [comment]
Yes i did the same,but still facing the issue.Also markers are not stable at the location.its keep on moving.
Please help me on this
@todoranalex @johnwedgbury

**Comment by todoranalex** [comment]
You can refer to #83, it kinda did the trick for me. 
Also, i think you should stop parse your marker list once you rendered it, because otherwise it will keep adding it's elements forever( possible cause of getting duplicates)

**Comment by pretivj** [comment]
Thanks for the reply,i tried the same way as you told.But markers are showing and then moving backward everytime. @todoranalex

**Comment by todoranalex** [comment]
I've observer the behaviour you are telling happening on some devices, but on others it works fine. Another solution would be to set a bigger refresh time for the anchors.( but this may cause as the anchors won't be in the best position the first time they appear :) )
